src/tasks/LibraryBuilder/Templates/CMakeLists.txt.template

```diff
@@ -37,6 +37,8 @@ if(TARGETS_ANDROID)
         libz.so
         log    
     )
+
+    set(CMAKE_C_FLAGS, "${CMAKE_C_FLAGS} -fPIC")
```

Comment:
nit: In Mono's cmake file we use add_definitions(-fPIC) to get it on both C and CXX flags, alternative is to do the same set for CXX flags as well to be complete.

Reply:
Was looking into that as well, we should do similar as we don on runtime builds and there its done for all platforms target by GCC (and Clang):

if(GCC)
  add_definitions(-g)  # TODO: should this really be on by default?
  add_definitions(-fPIC)
  add_definitions(-fvisibility=hidden)
  set(USE_GCC_ATOMIC_OPS 1)
endif()

There is also some logic passed down from mono-proj using -fpic when Android or LinuxBinonic and arm and arm64, so a little curious how that actually looks on the compile line when building that, if compiler will get both -fpic as well as -fPIC, that might be worth looking into.

Reply:
Added to the template cmake.